osu.Game.Rulesets.Osu.Tests/TestSceneSpinnerInput.cs

```diff
@@ -194,11 +196,11 @@ public void TestRewind()
             assertTotalRotation(3750, 810);
             assertTotalRotation(3500, 720);
```

Comment:
As discussed, it may be good to have these assert `(4000, 900)` again after each rewind step, to ensure forward playback is still correct.

Reply:
Have added this, fixed the clock interpolation and removed the test lenience as it is now precise \o/.